tests/test_coreg/test_affine.py

```diff
@@ -286,7 +286,7 @@ def test_deramping(self) -> None:
         assert np.abs(np.mean(periglacial_offset)) < np.abs(np.mean(pre_offset))
 
         # Check that the mean periglacial offset is low
-        assert np.abs(np.mean(periglacial_offset)) < 1
+        assert np.abs(np.mean(periglacial_offset)) < 2
```

Comment:
Why did this change?

Reply:
The subsample changed and the result is ~1.2 now. Let me check before...

Reply:
Before the test would have passed even with `< 0.01` :scream_cat:  ! OK, something is wrong with this now, I'll try to figure out why.

Reply:
Found it! The `inlier_mask` was never used in `Tilt`. This should get more consistent once we do the restructuration in Coreg :)